gremlin-core/src/test/java/org/apache/tinkerpop/gremlin/process/traversal/translator/GolangTranslatorTest.java

```diff
@@ -88,7 +88,7 @@ public void shouldTranslateGoNamedSteps() {
         assertEquals("g.V().Has(\"person\", \"name\", \"marko\").Where(gremlingo.T__.OutE().Count().Is(2).And(gremlingo.T__.Not(gremlingo.T__.InE().Count().Is(3))))", gremlinAsGo);
     }
 
-    // TODO AN-987: TraversalStrategy implementation in Gremlin-go
+    // TODO: TraversalStrategy implementation in Gremlin-go
```

Comment:
This is actually done.
```suggestion
```

Reply:
Added a tracker for the missing translator logic. The implementation of strategies is done but aren't able to be translated yet.